### ISSUE TITLE
Animate pilot and passengers

### ISSUE THREAD

**Issue by tigert** [Juanvvc/c172p-detailed#429]
i started this as a reply to #417 but this actually is a thing of its own.

It would be nice to animate the pilot and passengers properly, at minimum it would be nice if the pilot's left hand was on the yoke and right handle would grab and adjust the throttle and mixture when their values change.

   * pilot and passengers could animate their heads with some random jitter in a slow fashion (too fast will make everyone crazy ;)) and when you turn, their head movements could bias a bit to the inside of the turn.
   * pilot could scan instruments and look for traffic
   * if another aircraft is nearby, the passengers could look towards it and wave ;)
   * opening a window could cause the pilot/passenger lean their arm out ;)

I guess if there were bones in the models this could be done. I don't know how possible this currently is. Food for thought, however.

**Comment by gilbertohasnofb** [comment]
> at minimum it would be nice if the pilot's left hand was on the yoke and right handle would grab and adjust the throttle and mixture when their values change

The problem with the throttle and mixture would be that the animation of the model would be obviously delayed, but we must keep the change immediate (no lag for us piloting!). 

Currently the hands of the pilot are not very well positioned on the yoke, and her feet are also looking super bad in the rudder pedals. Fixing this would already make a huge difference IMO. Also, I think that animating only the arms and head for turns would suffice, as I am not a big fan of these models anyway (they creep me out! It's like taking Barbie and Ken for a ride) and so these ideas of waving to another plane and etc. do not really appeal to me, but that's just my 2 cents.

**Comment by tigert** [comment]
Yeah, maybe we could get new better looking folks as pilot and passengers, and could have them upstream in fgdata once they work well for us.

**Comment by gilbertohasnofb** [comment]
We could try to improve them indeed, but I think it's always so difficult to have good looking human models, they will always look very robotic and disturbing. See: https://en.wikipedia.org/wiki/Uncanny_valley :wink:

**Comment by Juanvvc** [comment]
There are bones in the models and they can be animated. In the walker model, animations are called "poses" and the process of creating them is slow. If I recall correctly, the walker includes a dialog box to configure poses.

If you want to check how one of these poses looks like, check this file: https://github.com/Juanvvc/c172p-detailed/blob/master/Models/Human/pose-pilot.xml Our pose for the copilot is similar, but the hands are on the lap and not on the yoke :)
